frontend/src/components/Graph/yAxisConfig.ts

```diff
@@ -48,8 +43,33 @@ export const getYAxisFormattedValue = (
 
 export const getToolTipValue = (value: string, format: string): string => {
 	try {
+		let decimalPrecision: number | undefined;
```

Comment:
```suggestion
		let decimalPrecision: number;
```
`undefined` is redundant here. It's implicitly understood when the value is not available.